.eslintrc.json

```diff
@@ -1,7 +1,7 @@
 {
   "extends": "eslint:recommended",
   "parserOptions": {
-    "ecmaVersion": "2017"
+    "ecmaVersion": 2018
```

Comment:
This reminds me that I should add a note to the README about which Node & webpack versions are explicitly supported. I'm only testing with Node >= 8 so this is fine. 👍